app/models/course.rb

```diff
@@ -35,6 +35,7 @@ class Course < ActiveRecord::Base
                                 source: :actable, source_type: Course::LessonPlan::Event.name
   has_many :material_folders, class_name: Course::Material::Folder.name, inverse_of: :course,
                               dependent: :destroy
+  has_many :forums, dependent: :destroy
```

Comment:
Not really, but the smaller the better. Here I think they can be separated. 
Might be a bit more work, but next time when we do a git bisect it would be 
easier to see how the idea changed over time.